Formula/o/openapi.rb

```diff
@@ -0,0 +1,25 @@
+class Openapi < Formula
+  desc "CLI tools for working with OpenAPI, Arazzo and Overlay specifications"
+  homepage "https://github.com/speakeasy-api/openapi"
+  # We use a git checkout since the build relies on tags for the version
+  url "https://github.com/speakeasy-api/openapi.git",
+      tag:      "v1.6.1",
+      revision: "fb2b971b296df25d463bb0fe826af97ead67b79d"
+  license "MIT"
+  head "https://github.com/speakeasy-api/openapi.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w"), "./cmd/openapi"
+  end
+
+  test do
+    assert_match "v#{version}", shell_output("#{bin}/openapi --version")
```

Comment:
your ldflags need update like this, https://github.com/speakeasy-api/openapi/blob/main/.goreleaser.yaml#L30-L33

Reply:
https://github.com/speakeasy-api/openapi/blob/main/cmd/openapi/main.go#L22

we actually support setting the version information both via our goreleaser build but also if you happened to install via go install as well. The homebrew build will use the go install code path which results in the version looking like:

```
➜  homebrew-core git:(add-openapi) /home/linuxbrew/.linuxbrew/Cellar/openapi/1.6.1/bin/openapi --version    
v1.6.1
Build: fb2b971
Built: 2025-08-29T01:13:32Z
```

after running `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source openapi`